### ISSUE TITLE
Initial Stress Test

### ISSUE THREAD

**Issue by edomamaru** [Energy-Visualizer/eviz#3]
- [x] Figure out why migrating the database throws a warning of no password in the service file
- [x]  Make a very basic page that displays data from the database.
- [x] Test Linear Algebra: extraction
    - [x] Get U and V matrices and store them. NumPY should be used to represent
    - [x] Compute shapes of U and V (make sure they're coming out correctly).
    - [x] Investigate data structures to hold RUVY matrices over multiple countries, years, etc.
- [x] Test Linear Algebra: computations
    - [x] Add U transposed and V and multiply them.
- [x] Research HTMX
- [x] Start stress testing Django's performance
    - [x] make a site with a button that does a bunch of queries.
    - [x] #6

**Comment by edomamaru** [comment]
- We do not need to migrate. 
- Table names should be defined in the meta data. It should also match exactly with the database table names.
- We defined the dataset column to be the primary key (was selected by random)
- We had to create environment variables that has the container paths that allows us to access the pgservice and the pgpass file. 
- We defined the basic database information in setting.py
- Check BASE_DIR to be the root directory of our project.
